src/harness/request_handler.py

```diff
@@ -127,12 +131,23 @@ def _Request(url, request, config, is_post_method):
     logging.info('POST Request to URL %s :\n%s', url, request)
   else:
     logging.info('GET Request to URL %s', url)
-  try:
-    conn.perform()
-  except pycurl.error as e:
-    # e contains a tuple (libcurl_error_code, string_description).
-    # See https://curl.haxx.se/libcurl/c/libcurl-errors.html
-    raise CurlError(e.args[1], e.args[0])
+
+  for attempt_count in range(MAX_REQUEST_ATTEMPT_COUNT):
+    try:
+      conn.perform()
+      error_message = ''
+      break
+    except pycurl.error as e:
+      # e contains a tuple (libcurl_error_code, string_description).
+      # See https://curl.haxx.se/libcurl/c/libcurl-errors.html
+      error_message += ' %s\n' % str(CurlError(e.args[1], e.args[0]))
+      time.sleep(REQUEST_ATTEMPT_DELAY_SECOND)
+    except Exception as e:
+      error_message += ' %s\n' % str(e)
+      time.sleep(REQUEST_ATTEMPT_DELAY_SECOND)
+  if error_message:
+    raise Exception(error_message.strip())
```

Comment:
Can you please raise the exception/error which was caught? I'm worried about interference with things like [this](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas_testcase.py#L239).

Unfortunately saving `e` and re-raising (i.e. `raise e`) interferes with the traceback, so the preferred option is to use just `raise` from inside the `except` since this gives the right behavior (example [here](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/testcases/WINNF_FT_S_SIQ_testcase.py#L1016)).